src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -68,7 +68,7 @@
 #include <uORB/topics/vehicle_trajectory_waypoint.h>
 
 #include "PositionControl.hpp"
-#include "Utility/ControlMath.hpp"
+#include "PositionControl/ControlMath.hpp"
```

Comment:
```suggestion
#include <ControlMath.hpp>
```

Reply:
True and PositionControl can be included the same way.